hw/top_darjeeling/lint/top_darjeeling_fpv_lint_cfgs.hjson

```diff
@@ -0,0 +1,153 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  flow: lint
+
+  // This is the primary cfg hjson for FPV linting. It imports ALL individual lint
+  // cfgs of the IPs FPV environments.
+  // This enables to run them all as a regression in one shot.
+  name: top_darjeeling_fpv_batch
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson"
+                     // tool-specific configuration
+                     "{proj_root}/hw/lint/tools/dvsim/{tool}.hjson"]
+
+  // Different dashboard output path for each tool
+  rel_path: "hw/top_darjeeling/fpv/lint/{tool}/summary"
+
+  use_cfgs: [{
+              name: alert_handler_esc_timer_fpv
+              fusesoc_core: lowrisc:opentitan:top_darjeeling_alert_handler_esc_timer_fpv
+              import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+              rel_path: "hw/ip/alert_handler/alert_handler_esc_timer_fpv/lint/{tool}"
+             }
+             {
+              name: alert_handler_ping_timer_fpv
+              fusesoc_core: lowrisc:opentitan:top_darjeeling_alert_handler_ping_timer_fpv
+              import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+              rel_path: "hw/ip/alert_handler/alert_handler_ping_timer_fpv/lint/{tool}"
+             }
+            {
+               name: prim_arbiter_ppc_fpv
+               fusesoc_core: lowrisc:fpv:prim_arbiter_ppc_fpv
+               import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+               rel_path: "hw/ip/prim/prim_arbiter_ppc_fpv/lint/{tool}"
+             }
+             {
```

Comment:
This seems a little silly to me. All the blocks below are identical between the top-levels (I think) and they probably shouldn't have been put in a file with "top_earlgrey" in the name before. Can we split things out into some "shared_fpv_lint_cfgs" list? (Maybe as a separate PR that lands before this one)

Reply:
Would adding that to the `import_cfgs` section work?

Reply:
I genuinely don't know! (And if it doesn't work, that might be a bug we should fix in dvsim...)